app/views/katello/api/v2/hosts_bulk_actions/module_streams.json.rabl

```diff
@@ -0,0 +1,7 @@
+object false
+
+extends "katello/api/v2/common/metadata"
+
+child @collection[:results] => :results do |_results|
+  extends 'katello/api/v2/module_streams/name_stream'
+end
```

Comment:
I had to repeat [this template](https://github.com/Katello/katello/blob/master/app/views/katello/api/v2/module_streams/name_streams.json.rabl), I wasn't able to extend it here, but maybe there is a way with rabl.